OJP/OJP_JourneySupport.xsd

```diff
@@ -59,11 +59,19 @@
 	</xs:complexType>
 	<xs:complexType name="ServiceSectionStructure">
 		<xs:annotation>
-			<xs:documentation>Properties of a service combined with a stop sequence interval. In many cases the elements about VEHICLE JOURNEY etc that are bundled in a service in OJP are enough for the whole Leg. However, in some cases enough of these element change within a single leg (usually a PT RIDE LEG), that e.g. the LINE, OPERATOR, FARE SECTION, COMMON SECTION changes. Possible cases are splitting/joining of trains, international trains at the border. Then it becomes necessary to tell at a StopPoint that the information has changed and this results in multiple ServiceSections. This implementation of ServiceSection is a view of some parts of JOURNEY PARTs.</xs:documentation>
+			<xs:documentation>Properties of a service combined with a stop sequence interval. In many cases the elements about VEHICLE JOURNEY etc that are bundled in a service in OJP are enough for the whole Leg. However, in some cases enough of these element change within a single leg (usually a PT RIDE LEG), that e.g. the LINE, OPERATOR, FARE SECTION, COMMON SECTION changes. Possible cases are splitting/joining of trains, international trains at the border. Then it becomes necessary to tell at a StopPoint that the information has changed and this results in multiple ServiceSections. This implementation of ServiceSection is a view of some parts of JOURNEY PARTs. Service Sections can exist in parallel (e.g. both coupled JOURNEYS from Bern to Kerzers, one part of the train then goes on to Murten the other to Neuenburg).</xs:documentation>
 		</xs:annotation>
```

Comment:
@ue71603 what is supposed to happen if the services are only coupled for part of the journey? To stick with your example, when you go from Bern to Neuenburg instead of Kerzers. Should the service to Murten be shown?

Reply:
Yes, it does, thanks for the explanation.